### ISSUE TITLE
Godot 4 won't respect animation list

### ISSUE THREAD

**Issue by SusanaHewitt** [godotengine/godot#64486]
### Godot version

4 alpha 14

### System information

Windows 11, GeForce GTX 1650

### Issue description

I have re-exported all my models as glbs, since Godot 4 unfortunately has trouble with daes, When I import my glbs and go to "advanced," then click on AnimationPlayer I'm allowed to slice my animations as was done in previous versions of Godot. But then Godot ignores the list, no matter what I do (it saves it and it can be seen every time I repeat the process of reimporting and going through the same steps). Yet all I ever get is a single animation named "All Animations" (as generated by the Babylon exporter in 3ds max).

By the way, I also can't find Quat in GDScript on Godot 4 alpha 14. The following lines don't compile:
var wrotation = Quat(global_transform.basis).slerp(Quat(wtransform.basis), rotation_speed)
global_transform = Transform(Basis(wrotation), global_transform.origin)

### Steps to reproduce

Import a glb and try slicing the animation.

### Minimal reproduction project

I can't send this specific project, but all you need is Godot 4 alpha 14 and a glb with multiple animations.

**Comment by tinmanjuggernaut** [comment]
Quat is called Quaternion.

As a work around for animation slicing, if you can get it into blender with separate animations, the workflow to Godot works well.

**Comment by SusanaHewitt** [comment]
Thanks for the Quaternion. They should put that in the docs. Also, thanks for the animations tip, but I don't want to make a hack. Much better to know that it's going to be fixed on alpha 15/beta 1.

**Comment by SusanaHewitt** [comment]
> Quat is called Quaternion.
> 
> As a work around for animation slicing, if you can get it into blender with separate animations, the workflow to Godot works well.

Does anyone know if this fix will be applied in the next release? Thanks.

**Comment by Calinou** [comment]
> Does anyone know if this fix will be applied in the next release? Thanks.

There is no ETA for fixing issues, especially the ones we don't know the cause of yet.

**Comment by SusanaHewitt** [comment]
> There is no ETA for fixing issues, especially the ones we don't know the cause of yet.

Wouldn't that be in some disabled block of code where it once enabled? And, if not, wouldn't animation parsing mostly be the same between formats?

**Comment by wenqiangwang** [comment]
Met with same issue here.

**Comment by wenqiangwang** [comment]
Might have found a clue: I noticed that the term used on Godot 4 UI for animation clips is now 'Slice'. So is import options saved into the .import file (e.g "slices/amount").

But it seems that in code, when we create the clips, we are still using the term 'clip'. I.e. we try to find import option "clips/amount" (see code [here](https://github.com/godotengine/godot/blob/d9e974cdb0ffa22053c81a214413c1d7b5d8cfcc/editor/import/resource_importer_scene.cpp#L959)). As a result, it cannot find the option and behaves there is not clips defined.

In short, I guess that could be caused by a partial rename, where we renamed 'clip' to 'slice' on UI, but not other places.

**Comment by wenqiangwang** [comment]
Verified it by replacing 'slice' to 'clip' in my test .import file and it seem Godot 4 not ignoring it anymore. 
I saw an error complaining *To create clips, animations must be named "default".*. I guess that is related to bug #40274, which is still not merged yet.

**Comment by SusanaHewitt** [comment]
Great news, thank you. Any chance the fix makes it to beta 3?
